### PR TITLE
SM-884: tweak edge-logs - add software version, tokenId, batteryVoltage, more

### PR DESCRIPTION
**Pull request by zakharenkodmytro**
- added TokenId to errors payload
- improved logging with options

### CODE REVIEW

File: internal/network/data_sender.go [hunk 1]
```diff
@@ -236,6 +236,9 @@ func (ds *dataSender) SendLogsData(data models.ErrorsData) error {
 	// sending the serial number of the device
 	if ds.unitID != uuid.Nil {
 		data.Device.UnitID = ds.unitID.String()
+		data.TokenID = ds.vehicleInfo.TokenID
```
Comment (JamesReate) [inline internal/network/data_sender.go:239]: data.VehicleTokenID ?

File: internal/models/network.go [hunk 2]
```diff
@@ -69,7 +69,8 @@ type SignalData struct {
 
 type ErrorsData struct {
 	CommonData
-	Device Device `json:"device,omitempty"`
+	Device  Device `json:"device,omitempty"`
+	TokenID uint64 `json:"vehicleTokenID"`
```
Comment (JamesReate) [inline internal/models/network.go:73]:
```suggestion
	VehicleTokenID uint64 `json:"vehicleTokenId"`
```

File: internal/loghooks.go [hunk 3]
```diff
@@ -98,3 +102,62 @@ func (h *LogRateLimiterHook) Run(e *zerolog.Event, _ zerolog.Level, msg string)
...
+}
+
+// logInfo logs an info message with the provided options.
+func logInfo(logger zerolog.Logger, message string, opts ...logOption) {
+	c := aplyOptions(opts)
+
+	logger.Info().Ctx(c).Msg(message)
+}
+
+func aplyOptions(opts []logOption) context.Context {
```
Comment (JamesReate) [inline internal/loghooks.go:147]:
```suggestion
func applyOptions(opts []logOption) context.Context {
```

File: internal/models/network.go (hunk 2 above)
Comment (zakharenkodmytro) [reply internal/models/network.go:73]: we have similar in here - https://github.com/DIMO-Network/edge-network/blob/c5a3f45ef4317ef3fdf46e0fa6b725dc2adffd88/internal/models/network.go#L139-L138

File: internal/loghooks.go (hunk 3 above)
Comment (zakharenkodmytro) [reply internal/loghooks.go]: fixed

File: internal/models/network.go (hunk 2 above)
Comment (zakharenkodmytro) [reply internal/models/network.go]: updated

File: internal/network/data_sender.go (hunk 1 above)
Comment (zakharenkodmytro) [reply internal/network/data_sender.go:239]: right

**Pull request merged**